README.md

```diff
@@ -14,19 +14,9 @@ Takeaway Challenge
 
```

Comment:
With regards to a README, the things a reviewer would love to see are:
- instructions on how to install the code
- instructions on how to run the code (a nice addition could be a code snippet or screenshot of example usage of the code)
- instructions on how to run the tests

Some nice to haves are:
- diagrams which represent the program. Perhaps in your planning you made class or sequence diagrams. These are nice to include as the reviewer will then have an idea of what the program looks like before diving into Ruby syntax
- descriptions of how you approached the challenge
- changes you would make if you had more time